src/layouts/AppContainer.tsx

```diff
@@ -9,37 +9,44 @@ import { ConnectionStatus, IWeb3State } from "reducers/web3Reducer";
 import store from "../configureStore";
 
 import * as arcActions from "actions/arcActions";
+import * as notificationsActions from "actions/notificationsActions";
 
 import CreateDaoContainer from "components/CreateDao/CreateDaoContainer";
 import CreateProposalContainer from "components/CreateProposal/CreateProposalContainer";
 import NoEthAccountContainer from "components/Errors/NoEthAccountContainer";
 import NoWeb3Container from "components/Errors/NoWeb3Container";
 import HomeContainer from "components/Home/HomeContainer";
+import Notification from "components/Notification/Notification";
 import ViewDaoContainer from "components/ViewDao/ViewDaoContainer";
 import HeaderContainer from "layouts/HeaderContainer";
 
+import { NotificationState } from "reducers/notificationsReducer";
 import * as css from "./App.scss";
 
 interface IStateProps {
   arc: IArcState;
   connectionStatus: ConnectionStatus;
   ethAccountAddress: string | null;
   web3: IWeb3State;
+  notifications: NotificationState;
 }
 
 const mapStateToProps = (state: IRootState, ownProps: any) => ({
   arc: state.arc,
   connectionStatus: state.web3.connectionStatus,
   ethAccountAddress: state.web3.ethAccountAddress,
   web3: state.web3,
+  notifications: state.notifications,
 });
 
 interface IDispatchProps {
   connectToArc: typeof arcActions.connectToArc;
+  closeNotification: (id: number) => any;
 }
 
 const mapDispatchToProps = {
   connectToArc: arcActions.connectToArc,
+  closeNotification: notificationsActions.dismissAlert,
```

Comment:
lets use the same name for the function here, so dismissAlert should be closeNotification